Makefile

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-NAMADA_VERSION := 0.23.0
+NAMADA_VERSION := 0.28.0
```

Comment:
I think this is `0.28.1`

Reply:
For the next testnet, should we switch to this branch?